src/dune_config/config.mli

```diff
@@ -60,10 +60,13 @@ val threaded_console : Toggle.t t
 (** The number of frames per second for the threaded console. *)
 val threaded_console_frames_per_second : [ `Default | `Custom of int ] t
 
+(** Control whether to show warnings for common typos in package dependencies *)
+val typo_warnings : Toggle.t t
+
 (** Before any configuration value is accessed, this function must be called
     with all the configuration values from the relevant config file
     ([dune-workspace], or [dune-config]).
 
     Note that environment variables take precedence over the values passed here
     for easy overriding. *)
-val init : (Loc.t * string) String.Map.t -> unit
+val init : (Loc.t * string) String.Map.t -> unit
```

Comment:
Please do not remove the trailing newlines. It creates changed lines where there are no changes actually.